docs/jumpstart/week-3.md

```diff
@@ -0,0 +1,418 @@
+let changableVar = 1;
+const unchangableVar = 2;
+
+changableVar = 3;
+unchangableVar = 4; // This will cause an error!!
+```
+
+It's important to keep in mind the scope of your variables. As we learn more about functions it'll become clearer, but the basic rule is that you can only use a variable in the same code block that you defined it or a code block that is nested in the block you defined it.
+
+## Data Types
+
+### Numbers and Strings
+
+Numbers can be specified as whole numbers or decimal numbers simply as you would type them normally.
+
+Strings are just text wrapped in quotes. You can use `"` or `'`. If you want to include a single or double quote in your string you can escape it with a backslash (`\`).
+
+#### Operations
+
+Javascript supports most basic operations that you may want to do with a number. You can add, subtract, multiply, and divide all with their basic operations `+ - * /`. Javascript also can do basic negation for a single number by specifying a negative sign `-` in front of a single number.
+
+```js
+const add = 1 + 2;
+const sub = 1 - 2;
+const mul = 1 * 2;
+const div = 1 / 2;
+const neg = -3;
+```
+
+Strings in Javascript can also be operated on with the addition symbol `+`. This will concat the two strings into one longer string.
+
+```js
+const hello = 'hello';
+const world = 'world';
+const helloWorld = hello + ' ' + world; // Equivilent to 'hello world'
+```
+
+Strings are essentially just a list of characters, so you'll often want to do similar things with strings that you might want to do with lists. 
+
+You can get the length of a string with their `.length` property. 
+ 
+You can also access any specific character of a string by using a square bracket `[]` and the character's index (position in the string). In Javascript, strings (like arrays) are *0-indexed*, meaning the first character has an index of 0, the next an index of 1, all the way until the last character's index which is the length minus 1.
+
+```js
+const str = 'Hello World!';
+const strLength = str.length; // Equal to 12 (Don't forget to count white space!)
+const letterH = str[0];
+const exclamation = str[str.length - 1]
+```
+
+
+### Booleans
+
+Booleans are either the value `false` or `true`. You will also hear the term *condition* which refers to a statement that evaluates to a boolean.
+
+#### Operations
+
+Javascript 2 operators for combining conditions: or `||` & and `&&`. 'or' will return true if either condition is true, and 'and' will only return true only if both conditions are true.
+
+You can also reverse a condition with the not operator `!`.
+
+```js
+const orOne = false || true;                  // This is true
+const orTwo = false || false || false;        // This is false
+const andOne = false && true && true;         // This is false
+const andTwo = true && true;                  // This is true
+const not = !true                             // This is false
+const combination = !true || !false && true;  // This is true
```

Comment:
Maybe show order of precedence (with and vs or) and using parentheses to group them.

Reply:
I don't think that's super important so I'm going to exclude it for now